site/ja/CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contribution Guideline
+
+## Communities
+
+* Slack
+  * Slack の #doc_translation チャンネルで議論をしています
+  * [TensorFlow User Group の公式ページ](https://tfug.jp/)から参加可能です
+* Google Groups
+  * [docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs)
+  * [docs-ja@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ja)
+
+## Directory Structure
```

Comment:
May want to clarify that `en/r2/` is `ja/beta/`
The translation directories (mostly) follow the URL structure of the site, like: https://www.tensorflow.org/beta

Reply:
I want to mention not only `en/r2/`.
For example, [`en/js/`](https://github.com/tensorflow/docs/tree/master/site/en/js) has a link to external repository.
In my understanding, we have to create new files and directories in [`ja/js/`](https://github.com/tensorflow/docs/tree/master/site/ja/js/tutorials).

Reply:
Yes, that's right, you will need to create `ja/js`, `ja/lite`, etc.
The difference is that there is no `en/beta`, that's just how it's imported into tensorflow.org. This sometimes confuses contributors but you should be fine since the directory already exists in `ja/`. And this should go away when we finally switch the site over to TF2 :)